src/devices/bosch.ts

```diff
@@ -1817,6 +1843,10 @@ const definitions: Definition[] = [
                 e.enum('motor_state', ea.STATE, Object.keys(stateMotor))
                     .withDescription('Shutter motor actual state '),
                 e.binary('child_lock', ea.ALL, 'ON', 'OFF').withDescription('Enable/Disable child lock'),
+                e.numeric('pushButtonHoldTime', ea.ALL).withUnit('s')
```

Comment:
Good point, i will test that in the evening but I dont know if light has a long press at all.

Reply:
Seems like you can set it, but it does not have any effect as far as i can see. I guess light does not have any long press. For shutter it is the difference between moving as long as you release or move completely.

Reply:
I will play around a bit to see if i can notify.

Reply:
renamed, you can set these tqo also on light, but no clue what this will do. My Button just ignored everything, my roocker on which i expected it to ignore went riot and i need to readd it so my system. So I suggest we ignore it for light. Base value on light btw is null. 
I assume it could get relevant on dimmer but i did not have a look into that yet. But thats a topic for another PR.